src/main/java/cz/centrum/haffner/SimpleTrainingDavid/REST/SimpleTrainingDavidApplication.java

```diff
@@ -1,9 +1,14 @@
 package cz.centrum.haffner.SimpleTrainingDavid.REST;
 
 import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
 
-@SpringBootApplication
+@Configuration
+@EnableAutoConfiguration
+@ComponentScan ("cz.centrum.haffner.SimpleTrainingDavid")
```

Comment:
Jaký byl důvod k tomu tady vyměnit
@SpringBootApplication

za

@Configuration
@EnableAutoConfiguration
@ComponentScan ("cz.centrum.haffner.SimpleTrainingDavid")
?

Reply:
Už jsem asi našel důvod - idea má nějaký potíž s component scanem a proto nejspíš některé komponenty podtrhávala červeně i když build procházel.
Líbí se mi, že sis poradil.
Pokud to chceš mít trochu čistější, můžeš ty 3 annotace vyměnit zpátky za jednu: ;-) 
**@SpringBootApplication(scanBasePackages = "cz.centrum.haffner.SimpleTrainingDavid")**

Reply:
předěláno na elegantnější řešení viz Tvůj popis ...